ui/app/src/services/content.ts

```diff
@@ -1257,9 +1257,10 @@ export function fetchChildrenByPaths(
         map(({ response: { items } }) => {
           const data = {};
           items.forEach(({ children, levelDescriptor, total, offset, limit, path }) => {
+            const totalWithDescriptor = levelDescriptor ? total + 1 : total;
```

Comment:
@rart the children API had a bug where the children count was considering the level-descriptor, now it has been fixed. But in the UI we include the descriptor (if exists) as part of the children (and considering it into the total). Since the API changed, we need to include the descriptor into the total for the UI to continue working properly (in PathNavs)